pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -204,6 +204,9 @@ func appendPlaceholderInterfacesToTheDomain(vmi *v1.VirtualMachineInstance, doma
 	if len(vmi.Spec.Domain.Devices.Interfaces) == 0 {
 		return domainSpec
 	}
+	if val := vmi.Annotations[v1.PlacePCIDevicesOnRootComplex]; val == "true" {
```

Comment:
Can you please add this information to the PR description?

Reply:
It is there already.
> Note: When "kubevirt.io/placePCIDevicesOnRootComplex" VMI annotation is used, no reservation is perform.